Gopkg.lock

```diff
@@ -10,6 +24,9 @@
 [[projects]]
   name = "github.com/Azure/azure-sdk-for-go"
   packages = [
+    "arm/authorization",
+    "profiles/2017-03-09/network/mgmt/network",
```

Comment:
As per my [other comment](https://github.com/Azure-Samples/azure-sdk-for-go-samples/pull/90/files#r167348498) this should end up being `github.com/Azure/azure-sdk-for-go/services/network/mgmt/2017-09-01/network`
and the next line should be changed too

Reply:
@sptramer can you run `dep ensure` again so these references are changed to services?